### DISCUSSION TITLE
Hardware Schematic/Board Files

### DISCUSSION THREAD

**Discussion by toneburst** [nordseele/hans#11] [Q&A]
Hi,

this looks great!!

I'd love to fork this project and create a narrow-format Eurorack module breaking out the USB sockets and MIDI IO on either/and DIN/TRS sockets.

Do you plan to release the schematic or board files for the hardware here?

Feel free to move this to a discussion, if you turn on that feature for the repo, as it's not strictly an "issue".\\

Keep up the great work!

**Comment by nordseele** [comment]
Hey, thanks for your message and for the tip about the "discussion", I didn't know this feature but this is great ;)
I've replied in details on Lines. The files: maybe, but the project is still in development and is likely to be ported to another board, probably not the right time for that. The schematics: https://www.midi.org/specifications-old/item/midi-din-electrical-specification (+ BOM for details) and the power input https://www.raspberrypi.org/documentation/hardware/raspberrypi/schematics/rpi_SCH_3b_1p2_reduced.pdf / The i2c line, have a look at the 16n faderbank schematics maybe, basically two pull-up resistors.

best,
